dev-python/ipcalc/ipcalc-1.99.0.ebuild

```diff
@@ -11,6 +11,8 @@ HOMEPAGE="https://pypi.python.org/pypi/ipcalc/"
 SRC_URI="mirror://pypi/i/${PN}/${P}.tar.gz"
 
 SLOT="0"
-LICENSE="MIT"
+LICENSE="BSD-2"
 KEYWORDS="amd64 x86"
 IUSE=""
+DEPEND="dev-python/six:=[${PYTHON_USEDEP}]"
```

Comment:
Why the `:=`?

Reply:
Hi, on other pr one of the change required are been use of := on DEPEND but maybe this is correct when package has more of one slot and then use of := is correct for identify all slots ? Can you explain when is correct use := ? thanks in advance

Reply:
:= is used when the package in question has slots or subslots, and you need to rebuild when dep is upgraded to higher slot/subslot. You can also use it when it's likely that the dep will have subslots in the future.

Since this is a python dep, rebuilding is irrelevant here.

Reply:
Ok, ty for clarification